core/src/main/scala/cromwell/core/WorkflowSourceFiles.scala

```diff
@@ -1,9 +1,11 @@
 package cromwell.core
 
+import better.files.File
 import wdl4s.{WdlJson, WdlSource}
 
 /**
   * Represents the collection of source files that a user submits to run a workflow
   */
 final case class WorkflowSourceFiles(wdlSource: WdlSource, inputsJson: WdlJson,
-                                     workflowOptionsJson: WorkflowOptionsJson)
+                                     workflowOptionsJson: WorkflowOptionsJson,
+                                     importsFile: Option[File] = None)
```

Comment:
I made this change and as I was doing it, I realized basically all WorkflowSourceFiles will require the imports path, because eventually they will be available through both server/run mode and thus it might make more sense to keep the optional file parameter, but not default it to None.